_posts/wip-reaching-out.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "2020-01-14: Interacting with the IAC Team"
+author: pmcmaw
+categories:
+  - team
+tags:
+  - community
+  - reviews
+  - modules
+  - windows
+  - communication
+---
+
+The IAC Team have decided to reach out to the community to make everyone aware of the best ways to get in contact with us. Whether this be to review a PR, highlight a nasty bug or really for any other reason. By doing this we hope to minimalise tasks falling through the cracks.
+
+## Monday is Community Day
+
+Yes, thats right! The whole team dedicates a full day to you every week. In this session we attempt to go over new issues, pull requests, releases and anything else module related.
+
+## Reaching out via Slack
+
+Weekly office hours on Monday 1.00pm and 5.00pm GTM alternating each week. Join #office-hours on our community-slack(puppetcommunity.slack.com). Between these hours the IAC team will be online and can chat about anything module related and much more!
+
+## Reaching out via Jira
+
+When you come across an issue with one of our supported modules and want to raise an issue, in order for to it to come directly to us there are a few things needed, this will ensure it lands on our teams plate and doesn't get lost in a large void of old and stale tickets.
+
+The values we would recommend setting are:
+
+* Project
+This will always need set to 'Infrastructure Automation Content (IAC)' if you want it to land with our team.
+
+* Issue Type
+Usually the community will be raising bugs against our supported modules however you can choose the descrption that best suits your problem.
+
+* Summary
+Just a quick simple summary of the topic you are looking to highlight to us.
+
+* Component
+This will quickly highlight to the team where the issue is being seen. This will help us triage your issue with a better understanding and highlight duplicate issues.
+
+* Description
+This is one of the most important sections, add in here as much details as you possibly can, don't hesistate to add sample manifests to help us reproduce your issue if your logging a bug. The more information here, the less likely we will need to go back and forth to reproduce and resolve your issue.
```

Comment:
should at least contain 
* Steps to reproduce
* results of running those steps
* Expected behaviour

maybe link to external resource on how to write a good bug report. We don't have to reinvent the wheel.